modules/ztp-generating-install-and-config-crs-manually.adoc

```diff
@@ -0,0 +1,146 @@
+
+. Generate the day-0 installation CRs by processing the modified `SiteConfig` CR `site-1-sno.yaml` by running the following command:
++
+[source,terminal,subs="attributes+"]
+----
+$ podman run -it --rm -v `pwd`/out/argocd/example/siteconfig:/resources:Z -v `pwd`/site-install:/output:Z,U registry.redhat.io/openshift4/ztp-site-generate-rhel8:v{product-version} generator install site-1-sno.yaml /output
+----
++
+.Example output
+[source,terminal]
+----
+site-install
+└── site-1-sno
+    ├── site-1_agentclusterinstall_example-sno.yaml
+    ├── site-1-sno_baremetalhost_example-node1.example.com.yaml
+    ├── site-1-sno_clusterdeployment_example-sno.yaml
+    ├── site-1-sno_configmap_example-sno.yaml
+    ├── site-1-sno_infraenv_example-sno.yaml
+    ├── site-1-sno_klusterletaddonconfig_example-sno.yaml
+    ├── site-1-sno_machineconfig_02-master-workload-partitioning.yaml
+    ├── site-1-sno_machineconfig_predefined-extra-manifests-master.yaml
+    ├── site-1-sno_machineconfig_predefined-extra-manifests-worker.yaml
+    ├── site-1-sno_managedcluster_example-sno.yaml
+    ├── site-1-sno_namespace_example-sno.yaml
+    └── site-1-sno_nmstateconfig_example-node1.example.com.yaml
+----
+
+. Optional: Generate just the day-0 `MachineConfig` installation CRs for a particular cluster type by processing the reference `SiteConfig` CR with the `-E` option. For example, run the following commands:
+
+.. Create an output folder for the `MachineConfig` CRs:
++
+[source,terminal]
+----
+$ mkdir -p ./site-machineconfig
+----
+
+.. Generate the `MachineConfig` installation CRs:
++
+[source,terminal,subs="attributes+"]
+----
+$ podman run -it --rm -v `pwd`/out/argocd/example/siteconfig:/resources:Z -v `pwd`/site-machineconfig:/output:Z,U registry.redhat.io/openshift4/ztp-site-generate-rhel8:v{product-version} generator install -E site-1-sno.yaml /output
+----
++
+.Example output
+[source,terminal]
+----
+site-machineconfig
+└── site-1-sno
+    ├── site-1-sno_machineconfig_02-master-workload-partitioning.yaml
+    ├── site-1-sno_machineconfig_predefined-extra-manifests-master.yaml
+    └── site-1-sno_machineconfig_predefined-extra-manifests-worker.yaml
+----
+
+. Generate and export the day-2 configuration CRs using the reference `PolicyGenTemplate` CRs from the previous step. Run the following commands:
+
+.. Create an output folder for the day-2 CRs:
++
+[source,terminal]
+----
+$ mkdir -p ./ref
+----
+
+.. Generate and export the day-2 configuration CRs:
++
+[source,terminal,subs="attributes+"]
+----
+$ podman run -it --rm -v `pwd`/out/argocd/example/policygentemplates:/resources:Z -v `pwd`/ref:/output:Z,U registry.redhat.io/openshift4/ztp-site-generate-rhel8:v{product-version} generator config -N . /output
+----
++
+The command generates example group and site-specific `PolicyGenTemplate` CRs for {sno}, three-node clusters, and standard clusters in the `./ref` folder.
++
+.Example output
+[source,terminal]
+----
+ref
```

Comment:
This structure doesn't look right. Note that group-du-xxx should be at the same level as common and example-sno. 

Also since three-node and standard-clusters are mentioned in line 133. They should be reflected as well.

```
$ ls -la ref/customResource/
total 20
drwxr-xr-x. 11 166536 yliu1  233 Sep 30 12:26 .
drwxrwxr-x.  3 166536 yliu1   28 Sep 30 12:26 ..
drwxr-xr-x.  2 166536 yliu1 4096 Sep 30 12:26 common
drwxr-xr-x.  2 166536 yliu1 4096 Sep 30 12:26 example-multinode-site
drwxr-xr-x.  2 166536 yliu1 4096 Sep 30 12:26 example-sno
drwxr-xr-x.  2 166536 yliu1  269 Sep 30 12:26 group-du-3node
drwxr-xr-x.  3 166536 yliu1   35 Sep 30 12:26 group-du-3node-validator
drwxr-xr-x.  2 166536 yliu1 4096 Sep 30 12:26 group-du-sno
drwxr-xr-x.  3 166536 yliu1   35 Sep 30 12:26 group-du-sno-validator
drwxr-xr-x.  2 166536 yliu1 4096 Sep 30 12:26 group-du-standard
drwxr-xr-x.  3 166536 yliu1   35 Sep 30 12:26 group-du-standard-validator
```

Reply:
Updated with:  
```
.Example output
[source,terminal]
----
ref
 └── customResource
      ├── common
      ├── example-multinode-site
      ├── example-sno
      ├── group-du-3node
      ├── group-du-3node-validator
      │    └── Multiple-validatorCRs
      ├── group-du-sno
      ├── group-du-sno-validator
      ├── group-du-standard
      └── group-du-standard-validator
           └── Multiple-validatorCRs
----
```